Adds first working graphql query

Vassyli opened the pull request:
This commit allows to query for realm information:
- coreVersion [string], fetches information from
  LotGD\Core\Game->gameVersion()
- crateVersion [string], static return value
- name [string], static return value

To run the server, use

```
php bin/console server:run
```

To write a realm query, call then the following url:

```
http://localhost:8000/?query=query%20RealmQuery{Realm{name,crateVersion,coreVersion}}
```

Vassyli committed:
Adds first working graphql query

This commit allows to query for realm information:
- coreVersion [string], fetches information from
  LotGD\Core\Game->gameVersion()
- crateVersion [string], static return value
- name [string], static return value

To run the server, use
```
php bin/console server:run
```

To write a realm query, call then the following url:
```
http://localhost:8000/?query=query%20RealmQuery{Realm{name,crateVersion,coreVersion}}
```

austenmc commented on app/config/config.yml:
```diff
@@ -0,0 +1,59 @@
+    - { resource: services.yml }
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
```
Not sure these fit this definition, but how crate/core config will be done is something we can iterate on.

Vassyli replied:
Not sure what you mean. This is the standard file after creating the project. I've put some default variables to parameters.yml, but I'm not using symfonys doctrine service anyway, lotgd\core manages the database for us. Real database connection parameters are in .env (which I need to read out manually on windows).

But yeah, configuration is an issue for itself.

austenmc commented on composer.json:
```diff
@@ -0,0 +1,79 @@
+        "sensio/framework-extra-bundle": "^3.0.2",
+        "incenteev/composer-parameter-handler": "^2.0",
+        
+        "lotgd/core": "dev-feature/buffs",
```
Let's try not to check this in. Set up it up to point to the lotgd/core repo on github w/ dev-master.

Vassyli replied:
I had to use this as long as my PR on lotgd/core was hanging. I'll be changing it to the normal github repository now.

austenmc commented on src/AppBundle/Resources/config/graphql/Realm.types.yml:
```diff
@@ -0,0 +1,18 @@
+Realm:
+    type: object
+    config:
+        description: "Common informations about the Realm"
```
"information" is plural already.

austenmc commented on src/AppBundle/Resources/config/graphql/Realm.types.yml:
```diff
@@ -0,0 +1,18 @@
+        fields:
+            name:
+                type: "String"
+                description: "The name of this realm."
```
Let's be consistent w/ periods. I prefer to have periods btw :)

austenmc commented:
Change the composer.json stuff but go ahead and commit this. I dont like  Real { coreVersion, crateVersion } b/c it's not extendable to learn more about the modules that are installed, which would also need versions, but for right now this is fine.

I think in the future we want something like a configuration connection (an array of configurations):

```
Realm {
  configuration {
    library # like lotgd/core
    name # like "Legend of the Green Dragon Core"
    description # some text here
    version # version string
  }
}
```

Vassyli commented:
Yeah, I just wanted to have something running and working for a first commit to see what we need to change in the core :)
I'll fix and push and "merge".

Vassyli committed:
Adds changes suggested in PR #1

Minor spelling fix and updated composer.json to use github repository
dev-master.

Closes #1
